controllers/metallb_controller_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	metallbv1beta1 "github.com/metallb/metallb-operator/api/v1beta1"
 	"github.com/metallb/metallb-operator/test/consts"
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Same comment I gave on the "changing the deps" commit.
This should be atomic, so I'd prefer to see a single "bump the code to ginkgo v2" commit that includes the deps